scaladoc/src/dotty/tools/scaladoc/renderers/HtmlRenderer.scala

```diff
@@ -207,7 +207,7 @@ class HtmlRenderer(rootPackage: Member, members: Map[DRI, Member])(using ctx: Do
       }
 
     val darkProjectLogoElem =
-      darkProjectLogo.flatMap {
+      darkProjectLogo.orElse(projectLogo).flatMap {
```

Comment:
Since we added this here, we can actually remove the `orElse(projectLogoElem)` from line 214. You're correct that it needs to go up here so that the id is unique and correctly hidden, but then we don't need the other `orElse`.